### ISSUE TITLE
packaging custom trained detectron2 model issue

### ISSUE THREAD

**Issue by Amir22010** [bentoml/BentoML#1713]
**Describe the bug**

`from bento_svc import CocoDetectronService`

`from detectron2.config import get_cfg`

`from detectron2 import model_zoo`

`from detectron2.checkpoint import DetectionCheckpointer`

`from detectron2.modeling import META_ARCH_REGISTRY`

`cfg = get_cfg()`

`cfg.merge_from_file(model_zoo.get_config_file("COCO-Detection/faster_rcnn_X_101_32x8d_FPN_3x.yaml"))`

`meta_arch = META_ARCH_REGISTRY.get(cfg.MODEL.META_ARCHITECTURE)`

`cfg.MODEL.ROI_HEADS.NUM_CLASSES = 12`

`cfg.MODEL.ROI_HEADS.SCORE_THRESH_TEST = 0.6`

`model = meta_arch(cfg)`

`model.eval()`

`device = "cpu"`

`model.to(device)`

`checkpointer = DetectionCheckpointer(model)`

`checkpointer.load("/content/model_final.pth")`

`metadata = {
    'device' : device
}`

`bento_svc = CocoDetectronService()`

`bento_svc.pack('model',model,metadata, model_zoo.get_config_file("COCO-Detection/faster_rcnn_X_101_32x8d_FPN_3x.yaml"))`

`bento_svc.save()`

but after running the 

`bentoml serve CocoDetectronService:latest --run-with-ngrok`

model is not able to predict the correct class index with accuracy score

[ERROR](https://slack-files.com/TK999HSJU-F025R4NTB46-b330443b00)
[ERROR1](https://slack-files.com/TK999HSJU-F025XSQT3EW-df626e92c8)

but when i edit the NUM_CLASS from 80 to 12 (no of custom classes) Line No 138 and 196 and WEIGHTS key value ("/content/model_final.pth") Line No - 268 which is the source of the trained model path in "model.yaml" artifacts

138: `NUM_CLASSES: 12`

196: `NUM_CLASSES: 12`

268: `WEIGHTS: /content/model_final.pth`

there is also a device issue problem - https://github.com/bentoml/BentoML/pull/1709 for this i have uploaded the "model.yml" file into the CocoDetectronService Artifacts, which contains to resolve model device issue.

`device: cpu`

and reran the `bentoml serve CocoDetectronService:latest --run-with-ngrok` 

It predict the correct class index with accuracy score

[SUCCESS](https://slack-files.com/TK999HSJU-F025R6AABSS-f0a1dd2ffe)

**Environment:**
 - OS: Linux
 - Python Version 3.7
 - BentoML Version 0.13.0

**Additional context**

**Comment by aarnphm** [comment]
I will close this for now. Everything should be fixed in the new release. Ping me if you still have problems.

**Comment by stvogel** [comment]
There is a 1.0 release in planning. That's great news, thanks @aarnphm 
Can I find some kind of roadmap anywhere or do you have a releasedate for 1.0 release in mind?
If you don't have an exact date ... will it be weeks or months?

**Comment by aarnphm** [comment]
I will ask the team about the roadmap and I will get back to you ASAP

**Comment by stvogel** [comment]
As @Amir22010  described you have to edit model.yml and replace the values in NUM_CLASSES and WEIGHTS and than it worked.
